src/components/Typography/Typography.module.scss

```diff
@@ -23,6 +23,9 @@
 
   &.onPrimary {
     color: var(--text-color-on-primary);
+    a {
+      color: var(--text-color-on-primary);
+    }
```

Comment:
shouldn't this be for every type?

Reply:
In other colors the link color should be link-color, but onPrimary and onInverted links should be in the same color as the other text (probably with underline)